patchwerk-sim.py

```diff
@@ -210,18 +253,44 @@ def run_simulation(tanks_list):
     
     tanks = [
         Tank(name='Bearly', max_health=11000, dodge_parry=0.25, mitigation=0.75),
-        Tank(name='Zug Zug', max_health=9500, dodge_parry=0.35, mitigation=0.725),
-        Tank(name='CTS', max_health=9500, dodge_parry=0.35, mitigation=0.725),
+        Tank(name='Zug Zug', max_health=9498, dodge_parry=0.35, mitigation=0.7),
+        Tank(name='CTS', max_health=9499, dodge_parry=0.35, mitigation=0.7),
     ]
 
+    # creates healers
+    # we use a dict as healer entities start with 1 rather than 0
+    # minimises confusions
+    healers = {}
+
+    # # 12 healer set up
+    # for healer_idx in range(1, 10):
+    #     assigned_tank_id = (healer_idx + 2) // 3 - 1
+    #     # test having different heals for different healers
+    #     # main_heal = 'h3'if healer_idx > 3 else 'gh1'
+    #     main_heal = 'h4'
+    #     healers[healer_idx] = Healer(entity=healer_idx, main_heal_used=main_heal, assigned_tank_id=assigned_tank_id)
+
+    # 13 healer set up
+    for healer_idx in range(1, 11):
+        if healer_idx <= 4:
+            assigned_tank_id = 0
+        elif healer_idx <= 7:
+            assigned_tank_id = 1
+        else:
+            assigned_tank_id = 2
+        # test having different heals for different healers
+        main_heal = 'h2'if healer_idx > 4 else 'h4'
+        # main_heal = 'h3'
+        healers[healer_idx] = Healer(entity=healer_idx, main_heal_used=main_heal, assigned_tank_id=assigned_tank_id)
```

Comment:
note for future: maybe we can make a config file that allows us to control each healer, its chosen heal, and its tank target